nixos/modules/services/x11/redshift.nix

```diff
@@ -114,7 +123,7 @@ in {
       partOf = [ "graphical-session.target" ];
       serviceConfig = {
         ExecStart = ''
-          ${cfg.package}/bin/redshift \
+          ${cfg.package}${cfg.executable} \
```

Comment:
```suggestion
          ${cfg.executable} \
```

Reply:
This _will_ break compatibility with the old version, so I'm not sure if what I'm proposing is the right way of doing it.

Reply:
I think that if we do `default = "${cfg.package}/bin/redshift";` then that would maintain backwards compatibility?

Reply:
I'm going to resolve these threads and not apply them for the reason @Infinisil mentioned. I think that having only one way to change the package and one way to change the executable seems like the best approach here.